src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/vedtaksvalidering.ts

```diff
@@ -44,66 +45,59 @@ const validerUtgiftsperioder = ({
-        //     };
-        // }
-
-        // if (!aktivitetstype && !opphørEllerSanksjon) {
-        //     return { ...utgiftsperiodeFeil, aktivitetstype: 'Mangler valg for aktivitetstype' };
-        // }
-
-        // if (!årMånedTil || !årMånedFra) {
-        //     return { ...utgiftsperiodeFeil, årMånedFra: 'Mangelfull utfylling av utgiftsperiode' };
-        // }
-
-        // if (!erMånedÅrEtterEllerLik(årMånedFra, årMånedTil)) {
-        //     return {
-        //         ...utgiftsperiodeFeil,
-        //         årMånedFra: `Ugyldig periode - fra (${årMånedFra}) må være før til (${årMånedTil})`,
-        //     };
-        // }
-
-        // const forrige = index > 0 && utgiftsperioder[index - 1];
-
-        // if (forrige && forrige.årMånedTil) {
-        //     if (!erMånedÅrEtter(forrige.årMånedTil, årMånedFra)) {
-        //         return {
-        //             ...utgiftsperiodeFeil,
-        //             årMånedFra: `Ugyldig etterfølgende periode - fra (${årMånedFra}) må være etter til (${forrige.årMånedTil})`,
-        //         };
-        //     }
-        // }
-
-        // if (barn.length < 1 && !opphørEllerSanksjon) {
-        //     return {
-        //         ...utgiftsperiodeFeil,
-        //         barn: ['Mangelfull utfylling - minst et barn må velges'],
-        //     };
-        // }
-
-        // if (barn.length > 0 && opphørEllerSanksjon) {
-        //     return {
-        //         ...utgiftsperiodeFeil,
-        //         barn: ['Skal ikke kunne velge barn ved opphør eller sanksjon'],
-        //     };
-        // }
+
+        if (!utgiftsperiode.periodetype) {
+            return { ...utgiftsperiodeFeil, periodetype: 'Mangler valg for periodetype' };
+        }
+
+        if (!utgiftsperiode.fra) {
+            return { ...utgiftsperiodeFeil, fra: 'Mangler fradato for periode' };
+        }
+
+        if (!utgiftsperiode.til) {
+            return { ...utgiftsperiodeFeil, til: 'Mangler tildato for periode' };
+        }
+
+        if (!erDatoEtterEllerLik(utgiftsperiode.til, utgiftsperiode.fra)) {
+            return {
+                ...utgiftsperiodeFeil,
+                til: 'Sluttdato (til) må være etter startdato (fra) for periode',
+            };
+        }
+
+        // TODO: Legge til sjekk av datoer mot forrige periode?
+
+        if (!utgiftsperiode.aktivitetstype) {
+            return { ...utgiftsperiodeFeil, aktivitetstype: 'Mangler valg for aktivitetstype' };
+        }
+
+        if (!utgiftsperiode.antallAktivitetsdager) {
+            return {
+                ...utgiftsperiodeFeil,
+                antallAktivitetsdager: 'Mangler valg av antall aktivitetsdager',
+            };
+        }
+
+        if (utgiftsperiode.barn.length === 0) {
+            return {
+                ...utgiftsperiodeFeil,
+                barn: ['Mangelfull utfylling - minst et barn må velges'],
+            };
+        }
+
+        if (!utgiftsperiode.utgifter) {
```

Comment:
Kanskje?
```suggestion
        if (!utgiftsperiode.utgifter || utgiftsperiode.utgifter < 0) {
```
Det valideres i backend men sikkert greit å ta med her også